app/Models/Litter/Categories/Alcohol.php

```diff
@@ -44,7 +22,7 @@ public function types ()
             'plasticAlcoholPackaging',
             'six_pack_rings',
             'alcohol_plastic_cups',
-            'alcoholOther'
+            'alcoholOther',
```

Comment:
You already reviewed the PR for these changes, and we agreed to keep the commas because they make for clean git diff views, so if we add a new item we'll get only a new line added, not two lines changed. This is a good practice used in most open-source projects. I suggest we keep them, are you sure we really need to remove them?